frontend/src/libs/components/meditation-tracks-list/meditation-track/styles.module.scss

```
@@ -0,0 +1,81 @@
+.track {
+  position: relative;
+  width: 100%;
+  max-width: 451px;
+  height: 177px;
+  margin: 0 auto;
+  border-radius: 30px;
+  box-shadow: 0 4px 23px 0 var(--violet-100);
+}
+
+.background-image {
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: block;
+  width: 37%;
+  height: 100%;
+  border-radius: 30px 0 0 30px;
+}
+
+.content {
+  position: absolute;
+  right: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 70%;
+  max-width: 85%;
+  height: 100%;
+  padding-right: 23px;
+  padding-left: 29px;
+  background-color: var(--white);
+  border-radius: 30px;
+}
+
+.info {
+  width: 52%;
+  height: 85px;
+  line-height: 20px;
+}
+
+.title {
+  margin: 0;
+  font-weight: 600;
+  font-size: 16px;
+}
+
+.duration {
+  display: inline-block;
+  margin-top: 8px;
+  padding: 6px 12px;
+  color: var(--blue-400);
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 16.8px;
+  text-align: center;
+  background-color: var(--blue-alpha-30);
+  border-radius: 14px;
+}
+
+.play-button {
+  display: block;
+  width: 48px;
+  height: 48px;
+  fill: var(--blue-300);
```

use color instead of fill, to make it work you need to pass currentColor to svg instead of color value (fill, border, stroke etc. depends on svg)
```suggestion
  color: var(--blue-300);
```

---

Do I need to set color instead of fill and set fil="currentColor" also for other sidebar icons and logo icons? Just in this case I see black color and styles for changing it were not imlemented

---

no, just for your icons, I will create another ticket for this

---

fixed